src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -106,102 +120,142 @@ export default function DownloadListTable() {
-                      >
-                        {item.fileName}
-                      </a>
-                    </td>
-                    <td>{item.projectName}</td>
-                    <td>{item.projectId}</td>
-                    <td>{addedOn}</td>
-                    <td>
-                      <UserCard
-                        size={'SMALL USER CARD'}
-                        ownerId={item.createdBy}
-                      />
-                    </td>
-                    <td>{createdOn}</td>
-                    <td>
-                      {item.fileSizeBytes &&
-                        calculateFriendlyFileSize(item.fileSizeBytes)}
-                    </td>
-                    <td>
-                      <button
-                        className={TESTING_TRASH_BTN_CLASS}
-                        onClick={() => {
-                          removeItem({
-                            fileEntityId: item.fileEntityId,
-                            versionNumber: item.versionNumber,
-                          })
-                        }}
-                      >
-                        <FontAwesomeIcon
-                          className="SRC-primary-text-color"
-                          icon="trash"
+        <>
+          <Table
+            striped={true}
+            responsive={true}
+            className="DownloadListTableV2"
+          >
+            <thead>
+              <tr>
+                <th>
+                  File Name
+                  <span>{showInteractiveSortIcon('fileName')}</span>
+                </th>
+                <th>
+                  Project
+                  <span>{showInteractiveSortIcon('projectName')}</span>
+                </th>
+                <th>
+                  SynID
+                  <span>{showInteractiveSortIcon('synId')}</span>
+                </th>
+                <th>
+                  Added On
+                  <span>{showInteractiveSortIcon('addedOn')}</span>
+                </th>
+                <th>
+                  Created By
+                  <span>{showInteractiveSortIcon('createdBy')}</span>
+                </th>
+                <th>
+                  Created On
+                  <span>{showInteractiveSortIcon('createdOn')}</span>
+                </th>
+                <th>
+                  Size
+                  <span>{showInteractiveSortIcon('fileSize')}</span>
+                </th>
+                {/* th below is made for trash can icon but holds no content */}
+                <th />
+              </tr>
+            </thead>
+            <tbody>
+              {allRows.map((item:DownloadListItemResult) => {
+                if (item) {
+                  const addedOn = moment(item.addedOn).format('L LT')
+                  const createdOn = moment(item.createdOn).format('L LT')
+                  return (
+                    <tr key={item.fileEntityId}>
+                      <td>
+                          {item.isEligibleForPackaging && <span
```

Comment:
Added an icon if the item is eligible for package (and a tooltip that says Eligible for packaging).